README.md

```diff
@@ -23,11 +23,14 @@ At this stage, `Colourblind8` will focus on developing settings for the three mo
 
 ### Functions Included In This Package
 
-- `set_deutera()`:
-  - This function implements a theme that makes plots accesible to people with deuteranopia. It modifies the colour of geometric objects (points, lines, etc) and the layout of the plot.
-- `set_prota()`
+- `plot_lines()`:
+   inputs: x variable,  y variable(s),  labels = None, palette = None, 
```

Comment:
I will add alpha as a parameter to `plot_line()` and `plot_scatter` as well.

Reply:
I can add it to just plot_scatter. It doesn't make sense for alpha with a line plot.

Reply:
I added it to the scatter plot!